src/passkit.cs

```diff
@@ -1586,6 +1638,22 @@ interface PKShareablePassMetadata {
 
 		[Export ("ownerDisplayName", ArgumentSemantic.Strong)]
 		string OwnerDisplayName { get; }
+
+		[Watch (8,0), iOS (15,0), Mac (12,0), MacCatalyst (15,0)]
+		[Export ("accountHash", ArgumentSemantic.Strong)]
+		string AccountHash { get; }
+
+		[Watch (8,0), iOS (15,0), Mac (12,0), MacCatalyst (15,0)]
+		[Export ("templateIdentifier", ArgumentSemantic.Strong)]
+		string TemplateIdentifier { get; }
+
+		[Watch (8,0), iOS (15,0), Mac (12,0), MacCatalyst (15,0)]
+		[Export ("relyingPartyIdentifier", ArgumentSemantic.Strong)]
+		string RelyingPartyIdentifier { get; }
+
+		[iOS (15,0), Mac (12,0), MacCatalyst (15,0)]
```

Comment:
missing `[Watch]`
this implies it was available in watchOS 2.0

Reply:
how do I determine the availability when the [docs don't indicate them](https://developer.apple.com/documentation/passkit/pkshareablepassmetadata/3850553-requiresunifiedaccesscapabledevi) and I can't find the relevant header file? (I'm assuming in this case that it's Watch (8, 0) but can never be too sure)

Reply:
@rachelkang if it does, look at the diff:
```
+                      requiresUnifiedAccessCapableDevice:(BOOL)requiresUnifiedAccessCapableDevice
 API_AVAILABLE(ios(15.0), watchos(8.0));
```
So, iOS 15 and watch 8 :) From https://github.com/xamarin/xamarin-macios/wiki/PassKit-iOS-xcode13.0-beta2

Reply:
oh, I totally missed that... they are so inconsistent!!! thanks 😆